en/messages.json

```diff
@@ -201,26 +216,22 @@
         "message": "You’re a Firefox Relay power user!",
         "description": "In the pop-up panel, when a free user has hit their mask limit, we tell them they're a power-user. DO NOT TRANSLATE \"Firefox Relay\"."
     },
-    "popupRegisterDomainHeadline": {
-        "message": "Get a custom domain for Relay email aliases",
-        "description": "DO NOT TRANSLATE \"Relay\"."
-    },
-    "popupRegisterDomainHeadline_mask": {
-        "message": "Get a custom subdomain for Relay email masks",
+    "popupGetDomainButton": {
+        "message": "Get a unique Relay email domain",
         "description": "DO NOT TRANSLATE \"Relay\"."
     },
-    "popupRegisterDomainButton": {
-        "message": "Register email domain"
-    },
-    "popupRegisterDomainButton_mask": {
-        "message": "Register email subdomain"
-    },
     "popupAliasesUsed": {
         "message": "Email aliases used"
```

Comment:
Looks like this string is replaced by the string below. Should we remove it all together?

Reply:
Removed in 47e6fc94f8ab2fdc620e148c6c1611ef1d1665e7